src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
+
+  getArea(): number;
 
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = Shape.triangle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Incorrect Data');
```

Comment:
<img width="1043" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/86789161/a39461ce-5796-4c69-95f2-8fa193c099ff">

Reply:
Same comment to all other error messages